src/utils.php

```diff
@@ -169,12 +169,13 @@ function isVendored(PhpDocument $document, \stdClass $composerJson = null): bool
  * @param \stdClass|null $composerJson
  * @param string         $uri
  * @param array          $matches
- * @return int
+ * @return string
  */
-function uriInVendorDir(\stdClass $composerJson = null, string $uri, &$matches): int
+function getPackageName(\stdClass $composerJson = null, string $uri): ?string
```

Comment:
I am pretty sure the return syntax you are using is only available in PHP7.1. That would be a breaking change, please omit the return type annotation